python/lsst/ts/standardscripts/auxtel/calibrations/power_on_atcalsys.py

```diff
@@ -0,0 +1,312 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["PowerOnATCalSys"]
+
+import asyncio
+import time as time
+
+import yaml
+
+from lsst.ts.idl.enums import ATWhiteLight
+from lsst.ts.observatory.control.remote_group import Usages
+
+
+from lsst.ts import salobj
+
+track_lamp_warmup = 60
```

Comment:
If this is supposed to be a constant, please, make is all caps, 

```
TRACK_LAMP_WARMUP = 60
```

But maybe best to have this as a class attribute instead, e.g.,. move it to the `__init__` method and make it `self.track_lamp_warmup`.